ksql-engine/src/test/java/io/confluent/ksql/structured/SchemaKStreamTest.java

```diff
@@ -172,7 +174,8 @@ public void init() {
         new KsqlConfig(Collections.emptyMap()),
         false,
         MockSchemaRegistryClient::new,
-        "test");
+        "test",
+        ProcessingLogContext.create());
```

Comment:
```suggestion
        processingLogContext);
```
Also in two more places below (in this same file).